client/signup/steps/design-type-with-atomic-store/index.jsx

```diff
@@ -131,7 +131,9 @@ class DesignTypeWithAtomicStoreStep extends Component {
 			designType,
 		} );
 
-		this.props.goToNextStep();
+		// If the user chooses `store` as design type, redirect to the `store-nux` flow
+		const nextFlow = designType === DESIGN_TYPE_STORE ? 'store-nux' : undefined;
```

Comment:
Is it possible to pass in the current flow name here instead of `undefined`? It would be more clear what is happening (without having to look into implementation of `goToNextStep`).